src/pages/Checkin/Checkin.vue

```diff
@@ -44,18 +44,20 @@
                       <br>
                       {{report.highlight}}
                     </p>
-                    <p @click="showData" v-if="canShowmore" class="text-primary btn">Read more..</p>
-                    <p
-                      class="textColor"
-                      v-if="canShowreason"
-                      :class="{not_completed : report.task_completed === false}"
-                    >{{report.task_not_completed_reason}}</p>
-                    <p
-                      class="text-warning"
-                      v-if="canShowreason"
-                      :class="{not_completed : report.task_completed === false}"
-                    >{{report.highlight_task_reason}}</p>
-                    <p @click="showData" v-if="canShowless" class="text-primary btn">Read less..</p>
+                    <span v-if="report.task_not_completed_reason !== '' || report.highlight_task_reason ==!'' ">
```

Comment:
*report.highlight_task_reason ==!''* ? this is supposed to be *report.highlight_task_reason !==''*

Reply:
report.highlight_task_reason ==!'' changed to report.highlight_task_reason !==''